### DISCUSSION TITLE
Broken private repos in v7.14.0

### DISCUSSION THREAD

**Discussion by hmnd** [pnpm/pnpm#5607] [General]
Wanted to make this discussion for better discoverability of the breaking change in [v7.14.0](https://github.com/pnpm/pnpm/releases/tag/v7.14.0) that affects private repo tokens in `.npmrc`.

If you have `//gitlab.com/api/v4/packages/npm/:_authToken` in your `.npmrc` and have suddenly started getting 404 errors when trying to `pnpm install`, you need to add a second config with a less specific url like `//gitlab.com/:_authToken`. 

Hope this helps someone else not spend hours on it! Wish there was greater visibility into this breaking change when running `pnpm install`.

**Comment by weyert** [comment]
Gitlab itself is broken too. I list all repos were I published packages to in my .npmrc and then I don’t have any problems. It’s a known issue at Gitlab and same issue exists for npm itself

**Reply by weyert** [reply]
https://gitlab.com/gitlab-org/gitlab/-/issues/352962 and https://gitlab.com/gitlab-org/gitlab/-/issues/334897
